tests/integration/testlib/statelib.py

```diff
@@ -117,13 +118,30 @@ def normalize(self):
         self._ignore_ipv6_link_local()
         self._sort_ip_addresses()
         self._sort_interfaces_by_name()
+        self.sanitize_ipv6()
 
     def match(self, other):
         return _state_match(self.state, other.state)
 
     def _sort_interfaces_by_name(self):
         self._state[INTERFACES].sort(key=lambda d: d['name'])
 
+    def sanitize_ipv6(self):
```

Comment:
How about naming this based on what it does specifically?
We already have specific "sanitize" changes like: `_ipv6_skeleton_canonicalization`.

Also note that this probably fits as a private method.

Reply:
using this method as 'public' I manage to use it from the statelib - check [0]. I also need to update the statelib to sanitize the addresses.

Having to kind of duplicate that in the statelib is what I least like ... But I couldn't come up with anything better that fit the existent design.

[0] - https://github.com/nmstate/nmstate/pull/545/files#diff-e631692ae7126c6af4eabc3757dbbde0R138